enterprise/reporting/label_set.go

```diff
@@ -60,6 +62,8 @@ func NewLabelSet(metric types.PUReportedMetric, bucket int64) LabelSet {
 		EventType:               metric.StatusDetail.EventType,
 		ErrorType:               metric.StatusDetail.ErrorType,
 		Bucket:                  bucket,
+		ErrorCode:               metric.StatusDetail.ErrorDetails.Code,
+		ErrorMessage:            metric.StatusDetail.ErrorDetails.Message,
```

Comment:
Don't we want to include these in `generateHash`?

Reply:
fixed